e2e/src/features/compile-invalid-contract.feature

```diff
@@ -11,8 +13,19 @@ Feature: Compile an invalid contract
       When I click the "button" with "Escrow Javascript" text
       Then I should be on the "javascript-editor" page
 
-      When I fill in the "JS Editor" input with "invalid contract"
+      When I find the line in the "monaco" editor containing 'const depositTimeout: Timeout = TimeParam("Payment deadline");'
+      And I press "Backspace" on the keyboard "4" times
 
       When I click the "button" with "Compile" text
       Then I should see a "button" with "Compiled" text
-      And the "button" with "Compiled" text should have "error" class
\ No newline at end of file
+      And the "button" with "Compiled" text should have "error" class
```

Comment:
Larry, why do we have this commented out section in here?

Reply:
Hey @paluh I commented this out because I could not get it to work  and thought I could work on this next. I guess I should add a comment explaining, or just remove it?